.github/workflows/publish_releasenotes_outside.yml

```diff
@@ -32,4 +37,4 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          ./.ci/update-github-page.sh ${{ github.event.inputs.version }}
+          ./.ci/update-github-page.sh ${{ inputs.version }}
```

Comment:
does it mean that we will not be able to run actions separately and manually as before ? or it works in both modes of execution.

it would be good to keep ability to run all release steps manually, because they do bunch of network activities and it is prone to fail. It would be good to have ability to continue release in manual way if something is failed in the middle.
We cannot rerun whole release from scratch, because we do commits and tags, close/create of milestones, publishing jars to maven (no delete abilities at all in maven central).

Reply:
> or it works in both modes of execution.

It does work in both modes of execution.

https://github.blog/changelog/2022-06-10-github-actions-inputs-unified-across-manual-and-reusable-workflows/

> Workflows triggered by workflow_dispatch and workflow_call can now access their inputs using the inputs context.

> Previously workflow_dispatch inputs were in the event payload. This made it difficult for workflow authors who wanted to have one workflow that was both reusable and manually triggered. Now a workflow author can write a single workflow triggered by workflow_dispatch and workflow_call and use the inputs context to access the input values.

> For workflows triggered by workflow_dispatch inputs are still available in the github.event.inputs context to maintain compatibility.

Reply:
Very good.